src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -142,7 +152,7 @@ if (ExecutionEnvironment.canUseDOM) {
   // IE9 claims to support the input event but fails to trigger it when
   // deleting text, so we ignore its input events
   isInputEventSupported = isEventSupported('input') && (
-    !('documentMode' in document) || document.documentMode > 9
+    !('documentMode' in document) || document.documentMode > 11
```

Comment:
while true, Microsoft has said that __a lot__ and then added another document mode :P it was more defensive than anything.